conf/messages

```diff
@@ -195,7 +195,11 @@ agent.error.business_accounting_method.empty                    = Select if your
 title_pattern.serviceName.govUk                                 = {0} - Use software to send Income Tax updates - GOV.UK
 agent_title_pattern.serviceName.govUk                           = {0} - Use software to report your client’s Income Tax - GOV.UK
 base.title.error                                                = Error: {0} - Use software to send Income Tax updates - GOV.UK
-agent_base.title.error                                           = Error: {0} - Use software to report your client’s Income Tax - GOV.UK
+agent_base.title.error                                          = Error: {0} - Use software to report your client’s Income Tax - GOV.UK
```

Comment:
What's this for? don't see it used anywhere in this PR

Reply:
It's just a realignment.